livedata/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -469,6 +469,7 @@ internal class ChannelControllerImpl(
         }
 
         newMessage.user = domainImpl.currentUser
+        newMessage.type = "regular"
```

Comment:
Is it always `regular` type? 
Could we use a constant value instead of this hardcoded string?

Reply:
Regular is a good default, ideally we would check for slash commands and do regular or ephemeral based on that.

Type should be an enum/sealed class in my opinion